src/helpers/error-message.helper.ts

```diff
@@ -87,29 +81,59 @@ export class ErrorMessageHelper {
         break;
       case AlgorithmErrorCode.WorkerNumberDuringNight:
         i = 0;
-        message = `Za mało pracowników w nocy w dniu <strong>${error.day + 1}</strong>`;
-        if (error.segments && error.segments[i][0] !== 22 && error.segments[i][1] !== 6) {
+        message = `Za mało pracowników w nocy`;
+        if (error.segments && (error.segments[i][0] !== 22 || error.segments[i][1] !== 6)) {
           message += ` w godzinach <strong>${error.segments[i][0]}-${error.segments[i][1]}</strong>`;
           while (error.segments[i + 1]) {
             i++;
-            if (error.segments[i][0] !== 22 && error.segments[i][1] !== 6) {
+            if (error.segments[i][0] !== 22 || error.segments[i][1] !== 6) {
               message += `, <strong>${error.segments[i][0]}-${error.segments[i][1]}</strong>`;
             }
           }
         }
-        message += `, potrzeba <strong>${error.required}</strong>, jest <strong>${error.actual}</strong>`;
+        message += `: potrzeba <strong>${error.required}</strong>, jest <strong>${error.actual}</strong>`;
         type = ScheduleErrorType.WNN;
         title = "date";
         if (error.day) {
           day += error.day;
         }
         break;
       case AlgorithmErrorCode.DissalowedShiftSequence:
-        message = `Niedozwolona sekwencja zmian dla pracownika <strong>${
+        const timeNeeded = ShiftHelper.requiredFreeTimeAfterShift(shifts[error.preceding]);
+        const earliestPossible = ShiftHelper.nextLegalShiftStart(shifts[error.preceding])[0];
+        const nextDay = ShiftHelper.nextLegalShiftStart(shifts[error.preceding])[1];
```

Comment:
Lepiej zrobić to za pomocą destrukcji, bo teraz to jest trochę mylące(łatwo nie zauważyć tego `[0]` na końcu)
Chodzi o coś takiego
```
const [eraliestPossible, isNextDay] = ShiftHelper.nextLegalShiftStart(shifts[error.preceding])
```